platform/email/src/main/java/org/motechproject/email/service/impl/EmailSenderServiceImpl.java

```diff
@@ -42,6 +43,7 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(EmailSenderServiceImpl.class);
 
     @Override
+    @Transactional
```

Comment:
In case we are unable to send mail, we throw an exception from this method, that will rollback the transaction and remove the added log.